### PR TITLE
Permissions language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

File: weblate/auth/permissions.py [hunk 1]
```diff
@@ -323,7 +323,7 @@ def check_unit_delete(user: User, permission: str, obj: Model):
         ):
             return Denied(
                 gettext(
-                    "Cannot remove terminology translation, remove source string instead."
+                    "Cannot remove terminology translation. Remove source string instead."
```
Comment (comradekingu) [inline weblate/auth/permissions.py:326]:
```suggestion
                    "Cannot remove terminology translation. Remove the source string instead."
```

File: weblate/auth/permissions.py [hunk 2]
```diff
@@ -281,13 +281,13 @@ def check_edit_approved(user: User, permission: str, obj: Model):
...
                 gettext(
-                    "Only reviewers can change approved strings, please add a suggestion if you think the string should be changed."
+                    "Only reviewers can change approved strings. Please add a suggestion if you think the string should be changed."
                 )
             )
     if isinstance(obj, Translation):
         component = obj.component
         if obj.is_readonly:
-            return Denied(gettext("The translation is read only."))
+            return Denied(gettext("The translation is read-only."))
```
Comment (comradekingu) [inline weblate/auth/permissions.py:290]:
```suggestion
            return Denied(gettext("This is a read-only translation."))
```
Better language, but not consistent form.

File: weblate/auth/permissions.py [hunk 3]
```diff
@@ -266,12 +266,12 @@ def check_edit_approved(user: User, permission: str, obj: Model):
...
         unit = obj
         obj = unit.translation
-        # Read only check is unconditional as there is another one
+        # Read-only check is unconditional as there is another one
         # in PluralTextarea.render
         if unit.readonly:
             if not unit.source_unit.translated:
                 return Denied(gettext("The source string needs review."))
-            return Denied(gettext("The string is read only."))
+            return Denied(gettext("The string is read-only."))
```
Comment (comradekingu) [inline weblate/auth/permissions.py:274]:
```suggestion
            return Denied(gettext("This is a read-only string."))
```

**Comment by nijel** [comment]
Due to recent cleanups in the templates, this pull request is not something we can easily merge due to massive conflicts. This is caused by the pull request being stale for a long time. This in turn is caused by its size and changing several unrelated files, what makes it challenging to review. Doing smaller pull requests would make the review and merging much easier, please consider it next time.

I will extract parts of this into separate pull requests for easier review.

**Comment by nijel** [comment]
All changes from this are now in separate pull requests.

**Pull request closed without merging**